openmrs-configs/configuration/concepts/disposition.csv

```diff
@@ -0,0 +1,9 @@
+Uuid,Void/Retire,Same as mappings,Fully specified name:en,Fully specified name:fr,Short name:en,Short name:fr,Description:en,Description:fr,Data class,Data type,Answers,Members,_version:1,_order:500
+efaf4399-601c-4479-aba2-21591a6eccae,,org.openmrs.module.emrapi:TRANSFER,Transfer Patient,Transfer Patient,Transfer Patient,Transfer Patient,,,Misc,N/A,,,,

```

Comment:
Do we need this file?
Doesn't the disposition work with out it?

Reply:
Alright, thanks